tests/erlang_tests/ceilfloatovf.erl

```diff
@@ -23,14 +23,24 @@
 -export([start/0]).
 
 start() ->
-    to_int(id(id([1.0e+65, 0]))).
+    to_int(id(id([1.0e+20, 0]))).
```

Comment:
just for my knowledge: why did you change this?

Reply:
I understand this test is actually a test of overflowing integers with AtomVM.
10^20 is sufficient to overflow, it's 17 hex digits.
With 65, the overflow is also a loss of precision of the float, so the number is not round, even with BEAM.